packages/synapse-react-client/src/components/SynapseHomepageV2/SynapseTrendingProjectsItem.tsx

```diff
@@ -0,0 +1,131 @@
+import { Box, Typography, SxProps } from '@mui/material'
+import React from 'react'
+import { calculateFriendlyFileSize } from '../../utils/functions/calculateFriendlyFileSize'
+import { useGetEntityHeader } from '../../synapse-queries'
+import { NavigateNext } from '@mui/icons-material'
+import { BackendDestinationEnum, getEndpoint } from '../../utils/functions'
+import { ReactComponent as EgressIcon } from '../../assets/homepage/egress.svg'
+
+import { ReactComponent as UsersIcon } from '../../assets/homepage/users.svg'
+export type SynapseTrendingProjectItemProps = {
+  rank: number
+  rowValues: (string | null)[]
+  entityIdColIndex: number
+  nUniqueUsersColIndex: number
+  egressSizeGbColIndex: number
+  isMobileView: boolean
+}
+
+const formatter = Intl.NumberFormat('en')
+export const trendingProjectsGridTemplateColumns: SxProps = {
+  gridTemplateColumns: {
+    xs: '100px auto 40px',
+    sm: '100px auto 170px 150px 40px',
+  },
+}
+
+export const SynapseTrendingProjectItem: React.FunctionComponent<
+  SynapseTrendingProjectItemProps
+> = ({
+  rank,
+  rowValues,
+  entityIdColIndex,
+  nUniqueUsersColIndex,
+  egressSizeGbColIndex,
+  isMobileView,
+}) => {
+  const entityId = rowValues[entityIdColIndex]
+  const egressSize = rowValues[egressSizeGbColIndex]
+  const { data: entityHeader } = useGetEntityHeader(entityId!, undefined, {
+    enabled: !!entityId,
+  })
+  const friendlyEgressSize = egressSize
+    ? calculateFriendlyFileSize(parseFloat(egressSize) * 1e9)
+    : ''
+
+  const userCount = rowValues[nUniqueUsersColIndex]
+  const friendlyUserCount = userCount
+    ? formatter.format(parseInt(userCount))
+    : ''
+  let backgroundColor = '#223549'
+  let borderRadius
+  switch (rank) {
+    case 1:
+      backgroundColor = '#447A74'
+      borderRadius = '12px 12px 0px 0px'
+      break
+    case 2:
+      backgroundColor = '#33575E'
+      break
+    case 3:
+      backgroundColor = '#294251'
+      break
+    case 10:
+      borderRadius = '0px 0px 12px 12px'
+      break
+  }
+  const egress = (
+    <Typography variant="body1">
+      <EgressIcon />
+      {friendlyEgressSize}
+    </Typography>
+  )
+  const users = (
+    <Typography variant="body1">
+      <UsersIcon />
+      {friendlyUserCount}
+    </Typography>
+  )
+  return (
+    <div
+      onClick={() => {
+        window.open(
+          `${getEndpoint(
+            BackendDestinationEnum.PORTAL_ENDPOINT,
+          )}Synapse:${entityId!}`,
+          '_target',
+        )
+      }}
+    >
+      <Box
+        sx={{
+          color: '#D7DEE4',
+          backgroundColor,
+          borderRadius,
+          '&:hover': {
+            color: 'white',
+            cursor: 'pointer',
+            filter: 'brightness(105%)',
+          },
+          p: '15px 0px',
+          display: 'grid',
```

Comment:
Consider using [subgrid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_grid_layout/Subgrid) if you think it might simplify this layout, but if grid-in-grid works well, please disregard

Reply:
I think this is grid working ok for now